### PR TITLE
Fix dist train with rmsprop

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/models/issues/1107

### CODE REVIEW

**Review by panyx0718 — COMMENTED** [review]
加个单测？

File: python/paddle/fluid/transpiler/distribute_transpiler.py [hunk 1]
```diff
@@ -1182,18 +1182,39 @@ def _append_pserver_ops(self, optimize_block, opt_op, endpoint,
...
+                if same_or_split_var(p.name, opt_op.input("Param")[0]):
+                    param_block = p
+                    break
+            return param_block
+
         for key in opt_op.input_names:
             if key == "Grad":
                 new_inputs[key] = merged_var
+            # For RMSProp optimizer
+            if key == "Moment" or key == "MeanSquare":
```
Comment (Yancey0623) [inline python/paddle/fluid/transpiler/distribute_transpiler.py]: `if` => `elif` ?

Comment (typhoonzero) [reply python/paddle/fluid/transpiler/distribute_transpiler.py]: Done

**Review by Yancey0623 — APPROVED** [review]
lgtm

**Pull request merged**